.github/workflows/deployment-develop.yml

```diff
@@ -0,0 +1,13 @@
+name: Vercel webhook
+on:
+  push:
+    branches: [develop]
```

Comment:
@amirhouieh Is it fine or must it be `staging`?

Reply:
This is fine, as long as you know which is what! So basicly you would use develop branch as for your staging/preview?

Reply:
Yes correct. Note I updated the README with instructions.